pandas/tests/io/test_s3.py

```diff
@@ -21,28 +20,31 @@ def test_streaming_s3_objects():
 
 
 @td.skip_if_no("s3fs")
-@pytest.mark.network
-@tm.network
-def test_read_without_creds_from_pub_bucket():
+@pytest.mark.single_cpu
+def test_read_without_creds_from_pub_bucket(s3_public_bucket_with_data, s3so):
     # GH 34626
-    # Use Amazon Open Data Registry - https://registry.opendata.aws/gdelt
-    result = read_csv("s3://gdelt-open-data/events/1981.csv", nrows=3)
+    result = read_csv(
+        f"s3://{s3_public_bucket_with_data.name}/tips.csv",
+        nrows=3,
+        storage_options=s3so,
+    )
```

Comment:
These are still accessing an external s3 bucket, yea? Not sure I understand the change here.

Reply:
Sorry, this used to be accessing a public bucket, but we have a mock public bucket that we run in our CI (via moto)